_content/av/back-to-nature_yuttadhamo.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Back to nature"
+authors:
+- yuttadhammo
+external_url: "https://tunein.com/podcasts/Religion--Spirituality-Podcas/Dhamma-Talks-(Part-3)-p2926505/?topicId=220379444"
```

Comment:
The preferred version is the cut on Drive, so no need to have an external url at all.